### ISSUE TITLE
Error with session_write_close()

### ISSUE THREAD

**Issue by jpmcc** [silverstripe/silverstripe-hybridsessions#31]
SilverStripe 4.0
PHP 7.1.12
Apache 2.4.18
MySQL 5.7.20

_Steps to reproduce:_

The module was installed using composer, as per supplied instructions. A session key was added to .env and used with the init method within site `_config.php` file:
```
$key = Environment::getEnv('SS_SESSION_KEY');
HybridSession::init($key);
```

On running `/dev/build/?flush=1`,  the following error occurs:
```
[Warning] session_write_close(): Session callback expects true/false return value
GET /dev/build?flush=1&flushtoken=8a78971e6e52ab96bcfb7ac46272a78f

Line 26 in /var/www/server/hybridsessions/src/Control/HybridSessionMiddleware.php
```

_Steps to fix:_

The error can be prevented by ensuring `HybridSession::write()` (line 113) returns a boolean. 

**_Original:_**
```
public function write($session_id, $session_data)
{
    if ($this->handlers) {
        foreach ($this->handlers as $handler) {
            if ($handler->write($session_id, $session_data)) {
                return;
            }
        }
    }
}
```

**_Change to:_**
```
public function write($session_id, $session_data)
{
    if ($this->handlers) {
        foreach ($this->handlers as $handler) {
            if ($handler->write($session_id, $session_data)) {
                return true;
            }
        }
    }
}
```

**Comment by jpmcc** [comment]
There is also an error:

```
[Warning] session_destroy(): Session callback expects true/false return value
GET /Security/logout?SecurityID=837fbecaecf08560ce34517aea6c021ca154f380

Line 288 in /var/www/server/vendor/silverstripe/framework/src/Control/Session.php
```

This can be resolved by ensuring `HybridSession::destroy()` (line 124), also returns a boolean.

**Comment by dhensby** [comment]
@jpmcc would you be able to submit a pull request with the two fixes you've identified?

**Comment by jpmcc** [comment]
I'll try and sort that out as soon as I can, just a little up against it on time at the moment. Will try and sort out in the next day or so.

**Comment by robbieaverill** [comment]
Fixed in #32 - thanks for reporting and for contributing the fix @jpmcc
